examples/src/components/GeoPushpinChartConfigurationPointsExample.jsx

```diff
@@ -0,0 +1,149 @@
+// (C) 2020 GoodData Corporation
+import React, { Component } from "react";
+import { GeoPushpinChart, Model } from "@gooddata/react-components";
+
+import "@gooddata/react-components/styles/css/main.css";
+
+import { projectId } from "../utils/fixtures";
+import { MAPBOX_TOKEN, cityCoordinatesUri, populationUri } from "../utils/fixturesGeoChart";
+
+const sizeMeasure = Model.measure(populationUri)
+    .format("#,##0.00")
+    .aggregation("sum")
+    .alias("Population");
+const locationAttribute = Model.attribute(cityCoordinatesUri).localIdentifier("location");
+const POINT_SIZE_OPTION = [
+    { value: "default", label: "default" },
+    { value: "0.5x", label: "0.5x" },
+    { value: "0.75x", label: "0.75x" },
+    { value: "normal", label: "normal" },
+    { value: "1.25x", label: "1.25x" },
+    { value: "1.5x", label: "1.5x" },
+];
+
+export class GeoPushpinChartConfigurationPointsExample extends Component {
+    constructor(props) {
+        super(props);
+
+        this.state = {
+            points: {},
+            togglePinSizes: false,
+        };
+    }
+    onLoadingChanged(...params) {
+        // eslint-disable-next-line no-console
+        return console.log("GeoPushpinChartConfigurationPointsExample onLoadingChanged", ...params);
+    }
+
+    onError(...params) {
+        // eslint-disable-next-line no-console
+        return console.log("GeoPushpinChartConfigurationPointsExample onError", ...params);
+    }
+
+    onZoomChanged(...params) {
+        // eslint-disable-next-line no-console
+        return console.log("GeoPushpinChartConfigurationPointsExample onZoomChanged", ...params);
+    }
+
+    onCenterPositionChanged(...params) {
+        // eslint-disable-next-line no-console
+        return console.log("GeoPushpinChartConfigurationPointsExample onCenterPositionChanged", ...params);
+    }
+
+    onPointSizeChange = event => {
+        const { id, value } = event.target;
+        this.setState(prevState => {
+            return {
+                ...prevState,
+                points: {
+                    ...prevState.points,
+                    [id]: value,
+                },
+            };
+        });
+    };
+
+    togglePinSizes = () => {
+        this.setState(prevState => {
+            return {
```

Comment:
we can return object directly, without 'return'
same as for other setState

Reply:
fixed